### PR TITLE
Move sanity_checks into custom action plugin

### PR DESCRIPTION
**Pull request by michaelgugino**
This commit moves sanity_checks tasks into a custom
action plugin that is only run against a single host.

This will result in a large reduction of tasks during initialization

### CODE REVIEW

File: roles/lib_utils/action_plugins/sanity_checks.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                   ('openshift_use_flannel', False),
+                   ('openshift_use_nuage', False),
+                   ('openshift_use_contiv', False),
+                   ('openshift_use_calico', False))
+
+
+def to_bool(var_to_check):
+    """Determine a boolean value given the multiple
+       ways bools can be specified in ansible."""
+    yes_list = (True, 1, "True", "1", "true", "Yes", "yes")
```
Comment (kwoodson) [inline roles/lib_utils/action_plugins/sanity_checks.py]: I think "on" and "On" and others are missing from this list.  'y' and 'Y' are also in the list.  

http://yaml.org/type/bool.html

Comment (michaelgugino) [reply roles/lib_utils/action_plugins/sanity_checks.py]: Nice.  Had no idea on was one.  Need to include a couple more varients as well.

**Pull request merged**